Sources/EpoxyCollectionView/Models/ItemModel/ItemModelBuilder.swift

```diff
@@ -0,0 +1,74 @@
+// Created by eric_horacek on 3/15/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+#if swift(>=5.4)
+/// A result builder that enables a DSL for building arrays of item models.
+@resultBuilder
+public struct ItemModelBuilder {
```

Comment:
could these be combined into something like:
```swift
@resultBuilder
public struct ArrayBuilder<T> {
  public typealias Expression = T
  public typealias Component = [T]
}
```
and then we can use this one builder for everything?
```

Reply:
Good idea, given that we're repeating the same code 6 times here I'll go ahead and make that change 👍